controllers/front/ProductController.php

```diff
@@ -328,6 +328,31 @@ public function initContent()
             // Assign attribute groups to the template
             $this->assignAttributesGroups($product_for_template);
         }
+
+        if (Tools::getValue('productajax')) {
+            ob_end_clean();
+            header('Content-Type: application/json');
```

Comment:
well, this was controlled via some global method in FrontController before when we used Tools::isSubmit('ajax'), maybe you should done this via `ajax=1&action=something`?